tests/func/test_data_cloud.py

```diff
@@ -68,6 +68,8 @@ def _should_test_gcp():
     if not os.path.exists(TestDvc.GCP_CREDS_FILE):
         return False
 
+    # NOTE: we copy because GOOGLE_APPLICATION_CREDENTIALS uses rel path.
+    #       This is only my guess and I don't know why won't we use absolute.
```

Comment:
Maybe then try using an absolute path?

Reply:
Maybe. I have no idea what's happening in staging, since I don't see all the env vars. Also, Travis is a pain to debug and this is blocking PR for me. So I am rather leave that to separate PR later.

Reply:
Then create an issue for it. No need to merge that second part of the comment into the upstream :slightly_smiling_face: